pkg/cas/content_addressable_storage_server_test.go

```diff
@@ -0,0 +1,88 @@
+package cas_test
+
+import (
+	"context"
+	"testing"
+
+	remoteexecution "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+	"github.com/buildbarn/bb-storage/internal/mock"
+	"github.com/buildbarn/bb-storage/pkg/blobstore/buffer"
+	"github.com/buildbarn/bb-storage/pkg/cas"
+	"github.com/buildbarn/bb-storage/pkg/digest"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestContentAddressableStorageServerBatchReadBlobsSuccess(t *testing.T) {
+	ctrl, ctx := gomock.WithContext(context.Background(), t)
+	defer ctrl.Finish()
+
+	digest1 := digest.MustNewDigest("ubuntu1804", "0000000000000000000000000000000000000000000000000000000000000001", 123)
+	digest2 := digest.MustNewDigest("ubuntu1804", "0000000000000000000000000000000000000000000000000000000000000002", 234)
+
+	request := &remoteexecution.BatchReadBlobsRequest{
+		Digests: []*remoteexecution.Digest{
+			digest1.GetPartialDigest(),
```

Comment:
Nit:

I'm always a big fan of this: https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html

Calls to functions like `.GetPartialDigest()` often make it harder to understand what's actually being passed around under the hood. It would have been perfectly fine to put `&remoteexecution.Digest{}` literals here instead of calling `.GetPartialDigest()`.